### PR TITLE
Peanuts, and Toasts, and More

### PR DESCRIPTION
**Pull request by VisVirific**
First of all, thank you Granodd for some of the sprites. They look awesome! And also thanks tgstation's AdipemDragon for the original sprites which Listerla ported over.

* Added various bread-related recipes!
  - Toast
  - Eggs in the basket
  - Peanut butter toast
  - PBJ Sandwich
  - Reuben Sandwich (Thanks Granodd!)
* Ports over Garlic Bread that the original port #12209 didn't bring
* Added Sauerkraut and Mayonnaise, for more authentic sandwhich shenaniganry (Thanks Granodd!)
* Added Roasted Peanuts, from which you can make Peanut Oil and Peanut Butter from
* Added new sprites for each addition, as well as a unique condiment jar for Cherry Jelly
**Sprites Added**

Recipes and general reagent sheaniganry can be changed with feedback.

EDIT: I FORGOT
* You can put Toasts in your mouth for the funsies.

### CODE REVIEW

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 1]
```diff
@@ -1920,6 +1920,16 @@
 	reagent_data = list(/decl/reagent/nutriment = list("bread" = 3, "cheese" = 3))
 	bitesize = 2
 
+/obj/item/reagent_containers/food/snacks/reubensandwich
+	name = "reuben sandwich"
+	desc = "A toasted sandwich packed with savory, meat and sour goodness!"
+	icon_state = "reubensandwich"
+	filling_color = "BF8E60"
```
Comment (mikomyazaki) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1927]:
```suggestion
	filling_color = "#BF8E60"
```

You could use color defines for all of these to make things easier. But probably none of the other hundreds of recipes do so... 🤷 

Check out `code/__defines/color.dm` for those.

**Review by mikomyazaki — CHANGES_REQUESTED** [review]

File: code/modules/reagents/reagent_containers/food/snacks.dm (hunk 1 above)
Comment (VisVirific) [reply code/modules/reagents/reagent_containers/food/snacks.dm:1927]: Unfortunately the ``filling_color`` is used for the custom recipes (sandwiches/breads/jelly/cereal/jawbreaker/sucker/candybars/donks/donuts/pies/kabob/waffles) to help distinguish what exactly a thing is made out of, so people tend to go with their own custom color choices.

It's hell

File: code/modules/reagents/reagent_containers/food/snacks.dm [hunk 2]
```diff
@@ -1931,6 +1941,39 @@
...
+	desc = "Egg in the basket, also known as <i>egg in a hole</i>, or <i>bullseye egg</i>, or <i>egg in a nest</i>, or <i>framed egg</i>, or..."
+	icon_state = "egginthebasket"
+	trash = /obj/item/trash/plate
+	filling_color = "#D9BE29"
+	reagents_to_add = list(/decl/reagent/nutriment = 2)
+	reagent_data = list(/decl/reagent/nutriment = list("toasted bread" = 2))
+	bitesize = 2
+
+/obj/item/reagent_containers/food/snacks/garlicbread
+	name = "garlid bread"
```
Comment (1Sparky1) [inline code/modules/reagents/reagent_containers/food/snacks.dm:1968]:
```suggestion
	name = "garlic bread"
```

Comment (VisVirific) [reply code/modules/reagents/reagent_containers/food/snacks.dm:1968]: GOD DAMN IT

**Review by mikomyazaki — APPROVED** [review]

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**